guides/common/modules/proc_deploying-a-custom-ssl-certificate-to-satellite-server.adoc

```diff
@@ -43,7 +43,8 @@ To update the certificates on a currently running Red Hat Satellite installation
     --certs-server-cert "_/root/satellite_cert/satellite_cert.pem_" \
     --certs-server-key "_/root/satellite_cert/satellite_cert_key.pem_" \
     --certs-server-ca-cert "_/root/satellite_cert/ca_cert_bundle.pem_" \
-    --certs-update-server --certs-update-server-ca
+    --certs-update-server --certs-update-server-ca \
+    --certs-update-all
```

Comment:
```suggestion
    --certs-update-server \
    --certs-update-server-ca \
    --certs-update-all
```

Reply:
See https://github.com/theforeman/foreman-documentation/pull/1678/files#r1071405936

Reply:
This can be simplified to:

```
    --certs-update-all
```

Reply:
Addressed :+1: